netlify.toml

```diff
@@ -16,3 +16,20 @@
 # Reference documentation
 # Optimization blog post: https://www.netlify.com/blog/2019/08/05/control-your-asset-optimization-settings-from-netlify.toml/
 # How to guide: https://docs.netlify.com/configure-builds/file-based-configuration/#post-processing
+
+# Redirect /quickstart/contour.yaml to the deployment that matches :latest.
+#
+# kubectl apply https://projectcontour.io/quickstart/contour.yaml
+[[redirects]]
+  from = "/quickstart/contour.yaml"
+  to = "https://raw.githubusercontent.com/projectcontour/contour/release-1.4/examples/render/contour.yaml"
+  status = 302
+
+# Redirect versioned quickstarts so that they can easily be referenced by
+# operators or for upgrade testing.
+#
+# kubectl apply https://projectcontour.io/quickstart/v1.4.0/contour.yaml
+[[redirects]]
+  from = "/quickstart/*/contour.yaml"
+  to = "https://raw.githubusercontent.com/projectcontour/contour/:splat/examples/render/contour.yaml"
```

Comment:
I assume netlify will do the usual input-sanitising here (preventing directory traversal and so on?)

Reply:
```
$ curl -v https://deploy-preview-2552--projectcontour.netlify.app/quickstart/../contour.yaml | head
...
< HTTP/1.1 404 Not Found
< Cache-Control: public, max-age=0, must-revalidate
< Content-Type: text/html; charset=utf-8
< Date: Wed, 27 May 2020 01:06:44 GMT
< Etag: 1555446584-ssl
< X-Robots-Tag: noindex
< Age: 24
< Content-Length: 2785
< Connection: keep-alive
< Server: Netlify
```

Not sure that it's really a problem anyway since the directory traversal would be on GitHub, which is public anyway.